packages/raystack/package.json

```diff
@@ -11,7 +11,11 @@
       "import": "./dist/index.js",
       "require": "./dist/index.cjs"
     },
-    "./index.css": "./dist/index.css"
+    "./v2": {
+      "import": "./dist/v2/index.js",
+      "require": "./dist/v2/index.cjs"
+    },
+    "./style.css": "./dist/style.css"
```

Comment:
Both v1 and v2 styles will be merged into a single file during build. This will not have conflicts as --rs- prefix is present for all v2 vars.